apps/frontend/components/Filters/SearchFilters.tsx

```diff
@@ -1,26 +1,52 @@
 import { Box, Button } from '@mui/material';
+import dayjs from 'dayjs';
 import { Dispatch, SetStateAction } from 'react';
-import { useForm } from 'react-hook-form';
+import { Controller, useForm } from 'react-hook-form';
 import { FormattedMessage } from 'react-intl';
 
+import { DateRange, DateRangeFilter } from './DateRangeFilter';
 import { DisasterFilter } from './DisasterFilter';
 
+interface SearchFormData {
+  disTyp: string;
+  dateRange: DateRange;
+}
+
 export const SearchFilters = ({
   setDisasterType,
 }: {
   setDisasterType: Dispatch<SetStateAction<string>>;
 }): JSX.Element => {
-  const { control, handleSubmit } = useForm({
-    defaultValues: { DisTyp: '1' },
+  const { control, handleSubmit } = useForm<SearchFormData>({
+    defaultValues: {
+      disTyp: '1',
+      dateRange: {
+        startDate: dayjs(new Date()),
+        endDate: dayjs(new Date()),
+      },
+    },
   });
 
-  const submitHandler = (data: { DisTyp: string }) => {
-    setDisasterType(data.DisTyp);
+  const submitHandler = (data: SearchFormData) => {
+    setDisasterType(data.disTyp);
   };
```

Comment:
i think this is a leftover from the way i was doing it before we added react-hook-form, we should be able to get rid of this function and all the state variables associated with it

Reply:
will resolve in backend ticket